src/pages/workspaces/workspace/workflows/DataStepContent.js

```diff
@@ -109,66 +90,36 @@ export default class DataStepContent extends Component {
           padding: '1rem 0.5rem', lineHeight: '1.5rem'
         }
       }, [
-        div({ role: 'radiogroup', 'aria-label': 'Select entities' }, [isSet ?
-          h(Fragment, [
-            div([
-              h(RadioButton, {
-                text: `Create a new set from all ${baseEntityTypeCount} ${baseEntityType}s`,
-                name: 'choose-set-components',
-                checked: isProcessAllAsSet,
-                onChange: () => this.setNewSelectionModel({ type: processAllAsSet, selectedEntities: {} }),
-                labelStyle: { marginLeft: '0.75rem' }
-              })
-            ]),
-            div([
-              h(RadioButton, {
-                text: `Create a new set from selected ${baseEntityType}s`,
-                name: 'choose-set-components',
-                checked: isChooseSetComponents,
-                onChange: () => this.setNewSelectionModel({ type: chooseSetComponents, selectedEntities: {} }),
-                labelStyle: { marginLeft: '0.75rem' }
-              })
-            ]),
-            hasEntityType && div([
-              h(RadioButton, {
-                text: 'Choose existing sets',
-                name: 'choose-set-components',
-                checked: isChooseSets,
-                onChange: () => this.setNewSelectionModel({ type: chooseSets, selectedEntities: {} }),
-                labelStyle: { marginLeft: '0.75rem' }
-              })
-            ])
-          ]) :
-          h(Fragment, [
-            div([
-              h(RadioButton, {
-                text: `Process all ${rootEntityTypeCount} rows`,
-                name: 'process-rows',
-                checked: isProcessAll,
-                onChange: () => this.setNewSelectionModel({ type: processAll, selectedEntities: {} }),
-                labelStyle: { marginLeft: '0.75rem' }
-              })
-            ]),
-            div([
-              h(RadioButton, {
-                text: 'Choose specific rows to process',
-                name: 'process-rows',
-                checked: isChooseRows,
-                onChange: () => this.setNewSelectionModel({ type: chooseRows, selectedEntities: {} }),
-                labelStyle: { marginLeft: '0.75rem' }
-              })
-            ]),
-            hasSet && div([
-              h(RadioButton, {
-                text: 'Choose existing sets',
-                name: 'process-rows',
-                checked: isProcessMergedSet,
-                onChange: () => this.setNewSelectionModel({ type: processMergedSet, selectedEntities: {} }),
-                labelStyle: { marginLeft: '0.75rem' }
-              })
-            ])
-          ])]),
-        !isProcessAll && !isProcessAllAsSet && div({
+        div({ role: 'radiogroup', 'aria-label': 'Select entities' }, [
+          isSet && div([
+            h(RadioButton, {
+              text: `Create a new ${rootEntityType} from selected ${baseEntityType}s`,
```

Comment:
Thinking out loud, I'm getting a little lost in the distinction between `rootEntityType` and `baseEntityType` here. Eg - it's not totally obvious to me which is lower, the root or the base.

My intuition would be that "root" would be below base", but that _seems_ like the opposite of what's happening here?

Reply:
Yeah, I wrestled with that a bit. The name `rootEntityType` is somewhat fixed, because that's what the method config calls it, and I think it's important to keep that the same. I considered 'subEntityType' but that seemed too obtuse.